plugin/pkg/admission/priority/admission.go

```diff
@@ -20,7 +20,7 @@ import (
 	"fmt"
 	"io"
 
-	schedulingv1beta1 "k8s.io/api/scheduling/v1beta1"
+	schedulingv1 "k8s.io/api/scheduling/v1"
```

Comment:
The head of the master branch must already have these changes. You should probably update your branch.

Reply:
Done I update my branch~ thank you~